addons/manifests/cni-webhook-manifests.yaml

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
@vijaykatam , do you think this is the right folder for putting webhook manifests?

Reply:
maybe create a `webhooks` subfolder under manifests?

Reply:
This should go into https://github.com/vmware-tanzu/tanzu-framework/tree/main/packages/management/addons-manager-v2/bundle/config/upstream and be templated with namespace. Also update https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/bundle/config/overlays/feature-gate-clusterbootstrap.yaml to add `@overlay/remove`

Reply:
Fixed. Now the webhooks manifests are moved under upstream and get generated only when `featureGates.clusterBootstrapController` is `false`